src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -92,31 +92,51 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
                          &_outPipe,
                          &_signalPipe,
                          &_piConhost,
+                         CREATE_SUSPENDED,
                          extraEnvVars));
 
-        _connected = true;
+        _hJob.reset(CreateJobObjectW(nullptr, nullptr));
+        THROW_LAST_ERROR_IF_NULL(_hJob);
+
+        // We want the conhost and all associated descendant processes
+        // to be terminated when the tab is closed. GUI applications
+        // spawned from the shell tend to end up in their own jobs.
+        JOBOBJECT_EXTENDED_LIMIT_INFORMATION jobExtendedInformation{ };

```

Comment:
nit: spaces between {} init braces or not?  Don't care, would just like to know what the preferred style is.

Reply:
The style is `{}` if there's no params and spaces if there are, like `{ foo, bar }`

Reply:
The style has come through and fixed it!